### DISCUSSION TITLE
How to switch between tabs using the keyboard

### DISCUSSION THREAD

**Discussion by luuhaa** [mikavilpas/yazi.nvim#532] [Q&A] [answered]
Just like the instruction of Yazi: use "[" and "]" to switch from tab to tab.
So how can I do this in yazi.nvim?

**Comment by mikavilpas** [comment, marked as answer]
No special config should be needed. Here is my config for reference https://github.com/mikavilpas/dotfiles/blob/0f61815e0068ccaf512fecc817f1b709620c21b0/.config/yazi/keymap.toml?plain=1#L152-L153

**Reply by luuhaa** [reply]
Great! Thx
